tests/unit/Http/Request/GetBestLanguageCest.php

```diff
@@ -27,6 +30,56 @@ public function httpRequestGetBestLanguage(UnitTester $I)
     {
         $I->wantToTest('Http\Request - getBestLanguage()');
 
-        $I->skipTest('Need implementation');
+        $store = $_SERVER ?? [];
+
+        $this->setNewFactoryDefault();
+        $request = $this->container->get('request');
+
+        $_SERVER = [
+            'HTTP_ACCEPT_LANGUAGE' => 'es,es-ar;q=0.8,en;q=0.5,en-us;q=0.3,de-de; q=0.9'
```

Comment:
Can we add some test cases with corrupt values? HTTP_ACCEPT_LANGUAGE can be manipulated by the client.